### PR TITLE
Create room function in admin 

### PR DESCRIPTION
**Pull request by orselane**

### CODE REVIEW

**Review by Mayukh-G — CHANGES_REQUESTED** [review]
Please remove the IN operator

File: src/db_api/database.py
Comment (Mayukh-G) [inline src/db_api/database.py:1]: The IN operator should not be added here

Comment (orselane) [reply src/db_api/database.py:1]: that's true, sorry I forgot to remove it

**Review by Mayukh-G — APPROVED** [review]
Looks good

**Pull request merged**